src/index.js

```diff
@@ -0,0 +1,40 @@
+import './style.scss';
+import Task from './modules/task.js';
+import Actions from './modules/actions.js';
+import Todos from './modules/todos.js';
+
+// Display tasks from localStorage
+document.addEventListener('DOMContentLoaded', Actions.displayTask);
+
+// Add a new task from input form
+const addBtn = document.querySelector('.add-btn');
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const todos = Todos.retrieveTask();
+  const todoText = document.querySelector('#add-item').value;
+  const completed = false;
+  const index = todos.length + 1;
+
+  if (todoText.trim() === '') {
+    alert('Please add a task');
```

Comment:
Consider manually styling an error message instead of using the window.alert()

Reply:
Thanks you 😀